jenkins-integration/jenkins-jobs/run_gatling_scenario.yml

```diff
@@ -0,0 +1,29 @@
+- project:
+    name: run_gatling_scenario
+    jobs:
+        - "run_gatling_scenario"
+
+- job-template:
+    name: run_gatling_scenario
+    parameters:
+    - string:
+        name: PUPPET_GATLING_SIMULATION_CONFIG
+        description: "The gatling scenario file to run"
+    - string:
+        name: PUPPET_GATLING_SIMULATION_ID
+        description: "An descriptive identifier for this run"
+    - string:
+        name: PUPPET_GATLING_MASTER_BASE_URL
+        description: "The URL of the puppet master SUT"
+
+    builders:
+        - sbt:
```

Comment:
Does the sbt plugin automatically take all of the job parameters and make them available to sbt as environment variables?